website/docs/guides/using-terraform-templates.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Using Terraform templates
+sidebar_position: 5
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Using Terraform templates <TierLabel tiers="enterprise" />
+
+This guide will show you how to use a template to create a Terraform resource in Weave GitOps Enterprise.
+
+### Pre-requisites
+- Install [Weave GitOps Enterprise](installation.mdx#weave-gitops-enterprise) with [TF-Controller installed](installation.mdx#tf-controller) and [TLS enabled](../configuration/tls.md).
+
+### 1. Add a template to your cluster 
+
+Add a Terraform `GitOpsTemplate` manifest, such as the one below, to your config repository at `./clusters/<cluster-name>/tf-template.yaml` or any path that Flux is configured to watch. Commit and push these changes. Once a template is available in the cluster, it can be used to create a Terraform resource, which will be shown in the next step.
+
+```yaml title="./clusters/my-cluster/tf-template.yaml"
+---
+apiVersion: clustertemplates.weave.works/v1alpha1
+kind: GitOpsTemplate
+metadata:
+  name: tf-template
+  namespace: default
+spec:
+  description: 
+    This is a sample WGE template that will be translated into a tf-controller specific template.
+  params:
+    - name: RESOURCE_NAME
+      description: Resource Name
+  resourcetemplates:
+    - apiVersion: infra.contrib.fluxcd.io/v1alpha1
+      kind: Terraform
+      metadata:
+        name: ${RESOURCE_NAME}
+        namespace: flux-system
+      spec:
+        interval: 1h
+        path: ./
+        approvePlan: "auto"
+        sourceRef:
+          kind: GitRepository
+          name: flux-system
+          namespace: flux-system
+```
+
+Verify that your template is in the cluster:
+```bash
+kubectl get gitopstemplates.clustertemplates.weave.works -A
+NAME                                AGE
+sample-wge-tf-controller-template   14m
+```
+
+If the template does not appear immediately, reconcile the changes with Flux:
+```bash
+flux reconcile kustomization flux-system
+► annotating Kustomization flux-system in flux-system namespace
+✔ Kustomization annotated
+◎ waiting for Kustomization reconciliation
+✔ applied revision main/e6f5f0c3925bcfecdb50bceb12af9a87677d2213
+```
+
+### 2. Create a Terraform resource
+A Terraform resource can be created from a template by specifying the template's name and supplying values to it, as well as your Weave GitOps Enterprise username, password, and HTTP API endpoint.
+```bash
+gitops add terraform --from-template sample-wge-tf-controller-template --set="RESOURCE_NAME"="name" --username=<username> --password=<password> --endpoint https://localhost:8000 --url <config-repo>
+```
+
+This will create a PR in your config repository with the manifest of the Terraform resource. Once the PR is merged, TF-Controller will apply the manifest, create the Terraform resource, and reconcile any changes to the Terraform manifest in your config repo! 
```

Comment:
Note: At this point, currently, the template will be created in a location not currently watched by Flux, so you have to move it manually: 
```
 mv clusters/management/clusters/tf-template.yaml clusters/kind/
```
This is a known issue and we're working to fix it :)